wiki/Installing-Oppia-Android.md

```diff
@@ -365,60 +374,55 @@ Please follow these steps to set up Oppia Android on your local machine.
 
    <img width="760" alt="Image" src="https://github.com/user-attachments/assets/5a26210f-865d-4048-b5a5-73379984bfb5" />
 
-7. In the above file, modify it so that it looks like below(copy and paste):
-   
-   ```
-   directories:
-      # Add the directories you want added as source here
-      # By default, we've added your entire workspace ('.')
-      .
-      
-   # Automatically includes all relevant targets under the 'directories' above
-   derive_targets_from_directories: false
-   
-   targets:
-   # If source code isn't resolving, add additional targets that compile it here
-      //:oppia_dev_binary
-      //app
-      //app/src/main/...
-      //domain/...
-      //model/...
-      //testing/...
-      //utility/...
-      //data/...
-      //scripts/...
-      
-   additional_languages:
-      # Uncomment any additional languages you want supported
-      # c
-      # dart
-      # kotlin
-      java
-      kotlin
-
-   # Please uncomment an android-SDK platform. Available SDKs are:
-      # android_sdk_platform: android-28
-      android_sdk_platform: android-29
-      # android_sdk_platform: android-30
-      # android_sdk_platform: android-31
-      # android_sdk_platform: android-32
-      # android_sdk_platform: android-33
-      android_sdk_platform: android-34
-   
-   shard_sync: true
-   sync_flags:
-      --local_ram_resources=HOST_RAM*.5
-      --discard_analysis_cache
+7. In the above file, modify it so that it looks like below (copy and paste):
    
-   java_language_level: 11
-   ```
+```
+directories:
```

Comment:
Let's replace this with an `import` statement per #5762 and specifically https://ij.bazel.build/docs/project-views.html#import. I think that will look like this:

```
import oppia-android.bazelproject
```

Reply:
Done. Also modified the import instructions.